src/localization/gps_linearization.py

```diff
@@ -63,11 +80,44 @@ def gps_callback(self, msg: NavSatFix):
         if not self.use_dop_covariance:
             msg.position_covariance = self.config_gps_covariance
 
-        self.last_gps_msg = msg
+        # print("using single callback")
```

Comment:
get rid of these prints, or make them rospy.logdebug

Reply:
fixed